source/retryable-reads/retryable-reads.rst

```diff
@@ -653,10 +637,33 @@ retry reads because the resilience benefit of retryable reads outweighs the
 minor risk of degraded performance. Additionally, any customers experiencing
 degraded performance can simply disable ``retryableReads``.
 
+Why did a previous version of the spec require that drivers retry operations only once?
+---------------------------------------------------------------------------------------
+
+The initial version of this specification mandated that drivers retry
+operations only once if the initial attempt failed with a transient error.
+However, this is not resilient to cascading failures in a cluster such as
+rolling server restarts during planned maintenance events. If the cluster is
+experiencing such transient failures, drivers expect the operation to succeed
+after some retries. If the cluster is actually in a permanently unhealthy state,
+though, we expect monitoring checks and background connection pool maintenance
+routines to start failing and servers to be marked unknown until server
+selection fails. Because server selection timeouts are not retryable, the
+operation will eventually fail.
+
+We make one exception for socket timeouts derived from the
+``socketTimeoutMS`` option. Such timeouts could occur due to transient
+network errors, so it’s useful to consider them retryable. However, they
+could also occur if the server requires more time than ``socketTimeoutMS`` to
+complete an operation. In this case, retrying indefinitely would result in an
+infinite retry loop. To maintain resiliency but avoid the undesirable
+infinite loop scenario, socket timeouts are only considered retryable once.
 
 Changelog 
 ==========
 
+2020-12-01: Change the retry policy to link to the client-side operations timeout specification
```

Comment:
This changelog entry is stale. It applies to a previous version where we kept the retry policy only in the new CSOT spec to avoid duplication between the retryable reads/writes specs but later decided it wasn't a good idea. I updated it to `Change the retry policy to retry indefinitely and special case socket timeouts`.